crates/contracts/build.rs

```diff
@@ -739,7 +815,7 @@ fn main() {
             .add_network_str(GOERLI, "0xE592427A0AEce92De3Edee1F18E0157C05861564")
             .add_network_str(SEPOLIA, "0xE592427A0AEce92De3Edee1F18E0157C05861564")
             .add_network_str(ARBITRUM_ONE, "0xE592427A0AEce92De3Edee1F18E0157C05861564")
-        // Not available on Gnosis Chain
+        // Not available on Gnosis Chain and Base
```

Comment:
Hmm, looks like base has a `UniversalRouter` that can be used for V2 and V3. Would be good to check if we can use it interchangeably.

Reply:
In order to use the `UniversalRouter` instead of `UniswapV3SwapRouter` they need to expose the same interface (or at least the parts we are using - `exactOutputSingle()`) which does not seem to be the case.